cw_monero/ios/Classes/monero_api.cpp

```diff
@@ -438,6 +467,11 @@ extern "C"
 
     const char *seed()
     {
+        std::string _rawSeed = get_current_wallet()->getCacheAttribute("cakewallet.seed");
+        if (!_rawSeed.empty())
```

Comment:
are we sure this won't return a null pointer if not available?

Reply:
Yes, according to the Monero `wallet2_api.h` doc-strings 
```
/*!
* \brief getCacheAttribute - return an arbitrary string attached to a wallet cache attribute
* \param key - the key
* \return the attached string, or empty string if there is none
*/
```